### PR TITLE
Data vbs militaer

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

**Comment by pfanguin** [comment]
jenkins test

**Comment by alcapat** [comment]
Jenkins test

**Comment by alcapat** [comment]
jenkins test

**Comment by alcapat** [comment]
Need it for deploy of 23.09.2015

File: chsdi/locale/de/LC_MESSAGES/chsdi.po [hunk 1]
```diff
@@ -3627,8 +3627,9 @@ msgstr "Einteilung geol. Generalkarte 200"
 msgid "ch.swisstopo.geologie-generalkarte-ggk200.metadata.autor"
 msgstr "Autor(en)"
 
+#. TODO
 msgid "ch.swisstopo.geologie-generalkarte-ggk200.metadata.ggk_nr"
-msgstr "Kartenblatt"
+msgstr "ch.swisstopo.geologie-generalkarte-ggk200.metadata.ggk_nr"
```
Comment (cedricmoullet) [inline chsdi/locale/de/LC_MESSAGES/chsdi.po]: Is it normal to remove translations ?

File: chsdi/locale/de/LC_MESSAGES/chsdi.po [hunk 2]
```diff
@@ -5672,8 +5729,9 @@ msgstr "Sektion"
 msgid "sektor"
 msgstr "Sektor"
 
+#. TODO
 msgid "sgd_nr"
-msgstr "InfoGeol-Nr."
+msgstr "sgd_nr"
```
Comment (cedricmoullet) [inline chsdi/locale/de/LC_MESSAGES/chsdi.po]: Is it normal to remove translations ?

**Comment by alcapat** [comment]
for "sgd_nr", it's a db clean. 
This key was replaced by some other translations key like ch.swisstopo.geologie-gisgeol-punkte.sgd_nr, ch.swisstopo.geologie-gisgeol-linien.sgd_nr, ch.swisstopo.geologie-gisgeol-flaechen-lt10km2.sgd_nr...) so i have removed this entry on empty.po (it's never use on other mako).

For the second one (ggk_nr), the key has changed on the db but not on the mako. So i have restored the key on the Db and translated again. 

Now translations files should be ok.
Would you like a squash for this commit ?

**Comment by alcapat** [comment]
jenkins test

**Comment by alcapat** [comment]
jenkins test

**Pull request merged**